index.html

```diff
@@ -53,5 +53,208 @@ <h3 class="hero-highlight-2">I’am a Software Developer</h3>
         </ul>
       </div>
     </section>
+    <div class="recent-work">
+      <p class="resent-work-txt">my recent works</p>
+      <hr />
+    </div>
+    <section class="grid-container">
+      <article class="cd-main-header">
+        <div class="placeholder-img"></div>
```

Comment:
[OPTIONAL] Please consider using the `<img/>` tag for displaying this `image` since it is not a `background` but a screenshot of the project meaning that it has value / is related to the content of this section. `CCS background` property is, as a rule, used for setting up backgrounds having only `styling` purposes instead of giving the content some value.